obs/preinstallimage-bios.sh.in

```diff
@@ -600,28 +600,25 @@ for F in /etc/tntnet/bios.d/*etn-ipm1-rest.xml ; do
     fi
 done
 
+# log configuration
+cp /usr/share/fty/examples/config/ftylog.d/10-ftylog.cfg /etc/fty/ftylog.cfg
+
 # Disable expensive debug logging by default on non-devel images
 mkdir -p /usr/share/fty/etc/default
 case "$IMGTYPE" in
     *devel*)
         echo "BIOS_LOG_LEVEL=LOG_DEBUG" > /usr/share/fty/etc/default/fty
         echo "BIOS_NUT_USE_DMF=true" >> /usr/share/fty/etc/default/fty
-        echo "log4cplus.rootLogger=DEBUG, console" > /etc/fty/ftylog.cfg 
+        sed -Ei "s|INFO|DEBUG|" /etc/fty/ftylog.cfg 
```

Comment:
I would be more strict 
sed -i "s|log4cplus.rootLogger=INFO|log4cplus.rootLogger=DEBUG|" ftylog.cfg